src/app/components/media/Similarity/MediaSimilarityBar.js

```diff
@@ -74,7 +74,6 @@ const MediaSimilarityBar = ({ projectMedia, setShowTab }) => {
               projectMediaDbid={props.project_media.dbid}
               confirmedSimilarCount={props.project_media.confirmedSimilarCount}
               hasMain={props.project_media.hasMain}
-              isSuggested={props.project_media.isSuggested}
```

Comment:
Ah, guessing you removed this because `MediaSimilarityBarComponent` doesn't care about the prop at all. I *think* this means we can remove the graphql property for `isSuggested` on line 24 as well??

Reply:
yeah, we used to put the count, so I removed this...I think you are right about line 24, will take a look

Reply:
if you remove it locally and it doesn't break anything it should be fine! I looked pretty hard down the component tree and didn't see anywhere in the code that uses it.